### PR TITLE
Better Dev mode

### PR DESCRIPTION
**Pull request by ericanderson**
The number of processes was getting out of hand. we now have a dev script that watches and triggers various build steps.

It is a bit more verbose than the old way but it also does things like run lint and tests which is a big win.

### CODE REVIEW

**Review by ssanjay1 — APPROVED** [review]
Approving to unblock, but left some questions for my own comprehension

File: packages/cli.cmd.typescript/src/cli.ts [hunk 1]
```diff
@@ -35,3 +35,11 @@ export async function cli(args: string[] = process.argv) {
     }
   }
 }
+//
```
[the remark this answers is not in the document]
Comment (ericanderson) [reply packages/cli.cmd.typescript/src/cli.ts:38]: Oops, i was triggering to make sure it builds right and forgot this. Thank you

File: packages/cli/tsconfig.json [hunk 2]
```diff
@@ -2,31 +2,11 @@
...
-      "path": "../shared.net"
-    },
-    {
-      "path": "../cli.cmd.typescript"
-    },
-    {
-      "path": "../cli.common"
-    }
-  ]
+  "references": []
```
[the remark this answers is not in the document]
Comment (ericanderson) [reply packages/cli/tsconfig.json:11]: To make everything work and be performant we are dropping the composite and references, which is more like how forge works than nucleus.

File: packages/foundry-sdk-generator/src/cli/foundrySdkGeneratorCli.ts [hunk 3]
```diff
@@ -36,3 +36,7 @@ export async function cli(args: string[] = process.argv) {
     console.log(e);
   }
 }
+//
```
Comment (ericanderson) [inline packages/foundry-sdk-generator/src/cli/foundrySdkGeneratorCli.ts:39]: fixme

File: tests/verify-fallback-package/tsconfig.json [hunk 4]
```diff
@@ -6,3 +6,7 @@
     "target": "es2018"
   }
 }
+
+//
```
Comment (ericanderson) [inline tests/verify-fallback-package/tsconfig.json:10]: fixme

**Comment by ericanderson** [comment]
We are going to yolo merge this too because we dont want chagnelogs for package.json dev changes.

**Pull request merged**